src/api/article/controller.js

```diff
@@ -80,6 +85,7 @@ export const create = async ({ body }, res, next) => {
       await article.save();
       data = serializeArticle(
         await article
+          .populate('author', '-_id firstName lastName email role active bio')
```

Comment:
What you think about moving strings like this `'-_id firstName lastName email role active bio'` to some constants in model to avoid repeating?